### DISCUSSION TITLE
LocalStack

### DISCUSSION THREAD

**Discussion by jaxxstorm** [pulumi/pulumi#7396] [General]
This discussion is designed to help users how to use Pulumi with [localstack](https://github.com/localstack/localstack) for testing

**Comment by delitescere** [comment]
I use the following script to create a `local` stack in Pulumi which uses LocalStack as the AWS endpoint https://gist.github.com/delitescere/366503e09df91820e015aa583e496a24#file-init-local-stack-sh

I use a few helper shell functions to interact with LocalStack based on how Pulumi creates resources https://gist.github.com/delitescere/366503e09df91820e015aa583e496a24#file-localstack-helpers-sh

_(note both these gists will get updated as time goes by)._
